srcpkgs/arc-theme/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'arc-theme'
 pkgname=arc-theme
-version=20180114
+version=20180715
 revision=1
 noarch=yes
 build_style=gnu-configure
 configure_script="./autogen.sh"
-hostmakedepends="automake pkg-config sassc"
-makedepends="gtk+3-devel"
+hostmakedepends="automake pkg-config sassc inkscape"
+makedepends="gtk+3-devel optipng"
```

Comment:
Don't you think `optipng` should be in the `hostmakedepends` too? It's just a binary.

Reply:
Fixed.